style.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@200;300;400&display=swap");
+
+:root {
+  --bg-color: #d6cda9;
+  --typewriter-speed: 5s;
+  --typewriter-char: 30;
+}
+
+body {
+  display: grid;
+  place-items: center;
+
+  background: var(--bg-color);
+  min-height: 100vh;
+  margin: 0;
+
+  font-family: "Source Code Pro", monospace;
+  text-align: center;
+}
+
+h1 {
+  position: relative;
+  font-size: clamp(1rem, 3vw + 1rem, 4rem);
```

Comment:
What does clamp do ?

Reply:
By using clamp, we can give a max and min limit to our font-size. 
Or you can say clamp give constraint to our font-size